dracut/30ignition/ignition-files.service

```diff
@@ -21,13 +18,13 @@ After=ostree-prepare-root.service
 After=initrd-switch-root.target
 Before=initrd-switch-root.service
 
+## NOTE(lucab): this was the original order in CL. This stage now runs
+##  after initrd-cleanup, and some behavior may be subtly different.
+
+#Before=initrd-parse-etc.service
```

Comment:
should this comment and line be moved back up to line 4 in the file where initrd-parse-etc.service was before?

Reply:
All of those can disappear once we are done with https://github.com/dustymabe/ignition-dracut/issues/20.